crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -835,6 +835,9 @@ en:
         swift_replicas:
           error: 'The number of replicas is bigger than the number of disks assigned for Swift.'
           help: 'Such configuration is not supported in Swift delivered with SOC 7. Adapt your Swift configuration before proceeding with the upgrade.'
+        lbaas_v1:
+          error: 'Old version of LBaaS is configured.'
+          help: 'Such configuration is not supported during the upgrade. Adapt your Neutron configuration to use LBaaS v2 before you continue.'
```

Comment:
"Such a configuration is no supported for the upgrade. Please switch to LBaaS v2 before you continue"